MetaMask/metamask-mobile
# chore: Blockaid remove unused unfair_trade reason type and minor cleanup

## **Description**

#### Blockaid code updates:
- remove unused unfair_trade reason type (see [Internal Thread](ref: https://consensys.slack.com/archives/C04D224QTLM/p1696953183067779))
- minor cleanup: update Reason const ordering
- fix REASON_DESCRIPTION_I18N_KEY_MAP typescript def

No UI/UX changes.

## **Manual testing steps**

1. go to test-dapp
2. test PPOM buttons
3. observe no changes

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

Fixes https://github.com/MetaMask/MetaMask-planning/issues/1484

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
